lib/read.js

```diff
@@ -239,7 +238,7 @@ class ReadSql extends AdapterRead {
         var timeField = this.timeField || 'time';
         var buckets = this.getBatchBuckets(from, to);
 
-        return Promise.map(buckets, (bucket, i, len) => {
+        return Promise.mapSeries(buckets, (bucket, i, len) => {
```

Comment:
You brought back the `mapSeries`! I still have no idea what it does, why is it good?